src/web/mjs/engine/Request.mjs

```diff
@@ -478,6 +478,13 @@ export default class Request {
         }
         delete details.requestHeaders['x-sec-fetch-site'];
 
+        // HACK: Imgur does not support request with accept types containing other mimes then images
+        //       => overwrite accept header to prevent redirection to HTML notice
+        if(/i\.imgur\.com/i.test(uri.hostname)) {
```

Comment:
its not just imgur
many madara sites also need this

Reply:
see https://github.com/manga-download/hakuneko/issues/2925#issuecomment-760201931
Aloalivn hosting on Aloalivn responding with a web page and not an image

Reply:
note: the reason it had to be implemented anyways was a "fix" for mangaku which broke after a couple days anyways

Reply:
The previous `Accept` header was completely wrong. HakuNeko was telling each website for any request that it only accept images and no text, json, html, ...
It is a miracle that it lasted that long until MangaKu decided to respond correctly (in an attempt to block HakuNeko of course).

If there are more hosts that prefer to respond with HTML instead with the image when HakuNeko says "i accept html, text, json, images, ..." then detection for image links needs to be improved, so HakuNeko will say i only accept images. As an alternative, affected connectors can still set the header in `_handleConnectorURI`.

Reply:
given _handleConnectorURI only deals with images, shouldn't the default behaviour be only accept images?

Reply:
the https://github.com/manga-download/hakuneko/pull/3082/commits/73efd906cb13f8857465c43725507713cb8ae75b commit seems to have fixed all known cases, but there is still edge cases where it wouldn't work and would require _handleConnectorURI to be altered

(the images not having an image extension)